components/tab/theme.css

```diff
@@ -0,0 +1,56 @@
+@import '@teamleader/ui-colors';
+
+.title-tab {
```

Comment:
Can you fix the styling when adding a `href` prop to the tab?
The same thing needs to be done when using a button element as for the tab, that looks a bit fucked up right now.
We need it to be possible to be a button for usability reasons. You can't use an anchor element without a href, because that's interactive with the keyboard (tab and enter/space I guess)

Reply:
Thanks for explaining why it's also needed

 :ok: by @InstaK on 30/11/2017 11:09:19